azure-functions/streaming-enabled/example_local.settings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "IsEncrypted": false,
+  "Values": {
+    "FUNCTIONS_WORKER_RUNTIME": "node",
+    "AzureWebJobsStorage": "",
+    "languageWorkers__node__arguments": "--inspect",
+    "NODE_OPTIONS": "-r newrelic",
+    "NEW_RELIC_LICENSE_KEY": "",
+    "NEW_RELIC_APP_NAME": "",
+    "NEW_RELIC_HOST": "staging-collector.newrelic.com",
```

Comment:
We may want to remove the `HOST` line and the `inspect` line in this general example.